### ISSUE TITLE
Can we set the "results_limit"?

### ISSUE THREAD

**Issue by lludol** [jakemmarsh/node-soundcloud#7]
Hello,

Do you know how we can set the "results_limit" of the request?
I know there is nothing about it in the documentation but I can't imagine that we can only receive 8 elements.

For example, if I use your module to search a music by putting "brumes" in the "q" parameter, I will have 8 results:

``` javascript
SC.get('/tracks', {
    q: 'brumes',
    results_limit: 20
}, (error, tracks) => {
    if (error) {
        console.log(error);
    } else {
        console.log(tracks.length); // 8
    }
});
```

If we look in the code of the SoundCloud website, we can see that:

```
https://api-v2.soundcloud.com/search/autocomplete?q=brumes&results_limit=20
```

I can set the "results_limit" and I will have 20 results.

How can I do that with your module?

**Comment by jakemmarsh** [comment]
Any options you pass as the second argument to `SC.get` will be passed as query parameters when making the request. This means the way you're doing `results_limit` in your example should work just fine. 

The example you give from the Soundcloud website is actually hitting a different endpoint (`/search/autocomplete` instead of `/tracks`), could that possibly be the reason for the discrepancy? Is your query only returning 8 tracks regardless of your `results_limit` because that's 100% of the possible results?

**Comment by lludol** [comment]
Thank you for your response.

I found what we have to send to the API (it's not written in the documentation !) :

``` json
{
  "q": "trackname",
 "offset": "0",
 "limit": 50
}
```

If for "q", we put "avicii" with an "offset" set to 0 and a "limit" to 50, we have 27 results.
If we set the "offset" to 1, we have now 28 results (the 27 + 1 new).
(now, I "just" have to understand why the limit/offset return random number of tracks)

I found that by looking at the API v2.
It (search/tracks/q=avicii) returns X results and an url to get the next X results by putting an offset and a limit inside the url...
